basic-spring-boot-tekton/.openshift/templates/build.yml

```diff
@@ -122,12 +124,15 @@ objects:
           - tag
           - "$(params.fromNamespace)/$(params.imageStream):$(params.tag)"
           - "$(params.toNamespace)/$(params.imageStream):$(params.tag)"
-      # not sure what to do here
       - name: verify-deployment
         image: quay.io/openshift-pipeline/openshift-cli:latest
-        command: ["sleep"]
+        command: ["oc"]
         args:
-          - "1"                             
+          - rollout
```

Comment:
@schen1 ; shouldn't 'rollout status' have the '--watch=true' param? or else this will just return not ready and continue. don't we want to wait until its deployed?

Reply:
@garethahealy It watches by default and the previous step (oc tag) triggers the deployment.
The description of the command is `Watch the status of the latest rollout, until it's done.`